main.tf

```diff
@@ -18,8 +19,6 @@ data "aws_route53_zone" "main" {
 ## ACM (AWS Certificate Manager)
 # Creates the wildcard certificate *.<yourdomain.com>
 resource "aws_acm_certificate" "wildcard_website" {
-  provider = aws.us-east-1 # Wilcard certificate used by CloudFront requires this specific region (https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html)
```

Comment:
Why removing the preferences to the `us-east-1` provider @edwinbdr ? This region is required for ACM if used by Cloudfront.

Reply:
I confirm it doesn't work w/o that region. 

<img width="1269" alt="Screenshot 2021-03-29 at 22 39 46" src="https://user-images.githubusercontent.com/10382023/112897251-ab3de880-90df-11eb-868e-d50942aee0d7.png">